scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -776,8 +780,18 @@ def cleanup_device_testing_artifacts(self):
 
         files_to_keep = self._get_binaries()
         files_to_keep.append(os.path.join('zephyr', 'runners.yaml'))
-
-        self.cleanup_artifacts(files_to_keep)
+        if self.testsuite.sysbuild:
+            files_to_keep.append('domains.yaml')
+            files_to_keep.append(os.path.join('zephyr', 'zephyr.signed.bin'))
+            files_to_keep.append(os.path.join('zephyr', 'zephyr.signed.hex'))
```

Comment:
Is this needed? I though we figure out which binaries are required based on runners.yaml https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/pylib/twister/twisterlib/runner.py#L784

Reply:
this can be optional, I think this can keep options open when do real flash

Reply:
what can be kept optional? runners.yaml defines which files you'll need for flashing. this in not optional. It seems in case of sysbuild runners.yaml has slightly different location than in regular cases, hence it don't find it with the current code. With the path updated for sysbuild, all binaries are properly added, including signed ones.

Reply:
I see you point, let me update